### PR TITLE
[DOCS-225, -217] Pentest Process

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Pentest Process | https://deploy-preview-317--cobalt-docs.netlify.app/platform-deep-dive/pentests/pentest-process/  | Rewrote Zendesk content describing the stages of the pentest program. The stages match the diagram on the main website: https://www.cobalt.io/pentest |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md [hunk 1]
```diff
@@ -3,10 +3,21 @@ title: "Pentest Process"
...
+Explore the stages of the Cobalt Pentest as a Service (PtaaS) program.
 {{% /pageinfo %}}
 
+![Lifecycle of the Cobalt pentest program](/deepdive/PentestProcessFlow.png "Lifecycle of the Cobalt pentest program")
+<br><br>
+
+| | Stage | Description |
+|---|---|---|
+| 1 | **Discover** | Prepare for the pentest engagement.<ul><li>Map the attack surface of your software.</li><li>Create an account on the Cobalt platform following our invitation.</li></li>  |
+| 2 | **Plan** | Plan, scope, and schedule your pentest. To understand your expectations, we'll arrange a call with you.<ul><li>See our [Getting Started](/getting-started/) guide to learn how to launch a pentest. If you need help, [contact us](/getting-started/#get-in-touch).</li><li>Prepare the environment for our pentesters, such as set up test credentials for them.</li><li>Alert the stakeholders in your organization about the upcoming pentest.</li><li>Once you've submitted the pentest, we'll assign pentesters based on your technology stack.</li></ul> |
```
Comment (yogipetkar) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md:19]: "To understand your expectations, we'll arrange a call with you."
We should get feedback from CS about this as they want to move away from calls as much as possible.

Comment (yogipetkar) [reply content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md:19]: check with Trang, Derek, Nathan and Andrew.

File: content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md [hunk 2]
```diff
@@ -3,10 +3,21 @@ title: "Pentest Process"
...
+| 3 | **Test** | Pentesters test your asset using various [pentest methodologies](/platform-deep-dive/pentests/pentest-process/methodologies/) and techniques.<ul><li>Pentesters share vulnerabilities that they discover in real time, in [Slack](/platform-deep-dive/collaboration/collaborate-on-pentests/#use-slack-for-communication) or in the Cobalt app. Ensure that at least one member of your team can answer questions or assist with troubleshooting. Learn more about [collaborating on a pentest](/platform-deep-dive/collaboration/collaborate-on-pentests/).</li><li>At this stage, you can start [remediating findings](/platform-deep-dive/pentests/findings/remediate-findings/) that pentesters discover.</li><li>The standard testing period is 14 days. It may vary depending on the [pentest scope](/getting-started/planning/#scope-the-pentest) and other factors.</li></ul>|
```
Comment (yogipetkar) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md:20]: "Pentesters share vulnerabilities that they discover in real time, in [Slack](/platform-deep-dive/collaboration/collaborate-on-pentests/#use-slack-for-communication) **and** in the Cobalt app."

Comment (yogipetkar) [reply content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md:20]: "Ensure that at least one member of your team can answer questions or assist with troubleshooting." Not sure what this means.

**Review by yogipetkar — CHANGES_REQUESTED** [review]

File: content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Pentests/Pentest Process/_index.md]: @yogipetkar I removed the part about arranging a call after confirming this with CSMs.

**Review by yogipetkar — APPROVED** [review]

**Pull request merged**